### PR TITLE
man page: Add all new heketi-cli commands and flags in man page

### PR DESCRIPTION
**Pull request by MohamedAshiqrh**
closes: #581 

Signed-off-by: Mohamed Ashiq Liyazudeen <mliyazud@redhat.com>

### CODE REVIEW

**Comment by MohamedAshiqrh** [comment]
@heketi/dev please review.

File: doc/man/heketi-cli.8 [hunk 1]
```diff
@@ -29,37 +29,37 @@ Command line program for Heketi
 \fBheketi\-cli cluster create\fP
 Create a cluster
     \fBExample\fP
-    $ heketi-cli cluster create
+    $ heketi\-cli cluster create
```
Comment (obnoxxx) [inline doc/man/heketi-cli.8]: These are fixes that should imho go into a separate patch: the don't fit under the commit message of "add all new commands".

File: doc/man/heketi-cli.8 [hunk 2]
```diff
@@ -72,32 +72,49 @@ Add new device to node to be managed by Heketi
...
 .PP
 .TP
 
-\fBheketi\-cli device delete <DEVICE-ID>\fP
+\fBheketi\-cli device delete <DEVICE\-ID>\fP
 Deletes a device from Heketi node
 
     \fBExample\fP
-    $ heketi-cli device delete 886a86a868711bef83001
+    $ heketi\-cli device delete 886a86a868711bef83001
```
Comment (obnoxxx) [inline doc/man/heketi-cli.8]: up to here it's actually unrelated fixes.

File: doc/man/heketi-cli.8 [hunk 3]
```diff
@@ -72,32 +72,49 @@ Add new device to node to be managed by Heketi
...
+.TP
+
+\fBheketi\-cli device enable <DEVICE\-ID>\fP
+Allows device to go online
+
+    \fBExample\fP
+    $ heketi\-cli device enable 886a86a868711bef83001
+.PP
+.TP
+
```
Comment (obnoxxx) [inline doc/man/heketi-cli.8]: new content

File: doc/man/heketi-cli.8 [hunk 4]
```diff
@@ -124,26 +141,72 @@ Add new node to be managed by Heketi
...
+\fBheketi\-cli node delete <NODE\-ID>\fP
 Deletes a node from Heketi management
 
     \fBExample\fP
-    $ heketi-cli node delete 886a86a868711bef83001
+    $ heketi\-cli node delete 886a86a868711bef83001
 .PP
 .TP
 
-\fBheketi\-cli node info  <NODE-ID>\fP
```
Comment (obnoxxx) [inline doc/man/heketi-cli.8]: fixes again up to here

**Comment by MohamedAshiqrh** [comment]
closing this as per request of split patch #583
